src/Sulu/Bundle/ContentBundle/DependencyInjection/SuluContentExtension.php

```diff
@@ -46,6 +46,12 @@ public function prepend(ContainerBuilder $container)
                             ],
                         ],
                     ],
+                    'resources' => [
+                        'pages_seo' => [
+                            'form' => ['@SuluContentBundle/Resources/config/forms/PageSeo.xml'],
+                            'endpoint' => 'get_page-seos',
```

Comment:
That's also weird... But I if I change the resource name to `page_seo`, then the URL will also be `/admin/api/page_seo`, which is suboptimal...